src/examples/zkapps/dex/arbitrary_token_interaction.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  isReady,
+  Mina,
+  AccountUpdate,
+  UInt64,
+  shutdown,
+  Token,
+} from 'snarkyjs';
+import { TokenContract, addresses, keys, tokenIds } from './dex.js';
+
+await isReady;
+let doProofs = true;
+
+let Local = Mina.LocalBlockchain({ proofsEnabled: doProofs });
+Mina.setActiveInstance(Local);
+let accountFee = Mina.accountCreationFee();
+
+let [{ privateKey: userKey, publicKey: userAddress }] = Local.testAccounts;
+let tx;
+
+console.log('-------------------------------------------------');
+console.log('TOKEN X ADDRESS\t', addresses.tokenX.toBase58());
+console.log('USER ADDRESS\t', userAddress.toBase58());
+console.log('-------------------------------------------------');
+console.log('TOKEN X ID\t', Token.Id.toBase58(tokenIds.X));
+console.log('-------------------------------------------------');
+
+// compile & deploy all 5 zkApps
+console.log('compile (token)...');
+await TokenContract.compile();
+
+let tokenX = new TokenContract(addresses.tokenX);
+
+console.log('deploy & init token contracts...');
+tx = await Mina.transaction(userKey, () => {
+  // pay fees for creating 2 token contract accounts, and fund them so each can create 1 account themselves
+  let feePayerUpdate = AccountUpdate.createSigned(userKey);
+  feePayerUpdate.balance.subInPlace(accountFee.mul(1));
```

Comment:
Is there a reason for multiplying it times one? Probably not, just curious tho

Reply:
no there isn't :D it was copied from another place where it was multiplied by 2 I guess